README.md

```diff
@@ -1 +1,15 @@
-# liq-bot
+# Typescript Liquidation Bot for HubbleProtocol by Lmvdzande
```

Comment:
Some explanation about why we need both tryLiquidate and clearLiquidation gains could help

Reply:
Can do!